firebase.js

```diff
@@ -0,0 +1,22 @@
+// firebase.js
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+import { getFirestore } from "firebase/firestore";
+import { getAnalytics } from "firebase/analytics";
+
+// Your web app's Firebase configuration
+const firebaseConfig = {
+  apiKey: "AIzaSyBcEsnGJGryp2saF_Yx22ZY65oU1Yr1sKY",
```

Comment:
Ideally you should not push creds to git
should look like import.meta.env.VITE_FIREBASE_API_KEY, and for the rest of params